### ISSUE TITLE
Obsoletion request: cell surface bile acid receptor signaling pathway	

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#26958]
[cell surface bile acid receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038184)
has the same meaning as [G protein-coupled bile acid receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038182)

**Comment by raymond91125** [comment]
Yes, they are related as the MF is part_of BP. Which one should be obsoleted?

**Comment by pgaudet** [comment]
'[cell surface bile acid receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038184)' 
G protein signaling is more specific 

Thanks, Pascale

**Comment by raymond91125** [comment]
Sorry, I don't understand. You want to merge the pathway, a BP into the receptor activity, an MF? They don't have the same meaning. As of now, MF is part of BP, which seems to be correct.
Although at the moment the only annotations to the BP "GO:0038184 cell surface bile acid receptor signaling pathway" are the receptors, there is more known about the pathway, e.g. PMID:36409000. If we obsoleted it now, we may have to resurrect it soon.

**Comment by pgaudet** [comment]
Hi @raymond91125 
Sorry I pasted the wrong terms. We need to fix both MF and BP here

**MF**

- [ ] obsolete grouping class [bile acid receptor activity](http://purl.obolibrary.org/obo/GO_0038181)
 with CONSIDER [G protein-coupled bile acid receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038182) and  ([lithocholic acid receptor activity](http://purl.obolibrary.org/obo/GO_0038186)). 
- [ ] add synonym 'farnesoid X receptor activity' and  'farnesoid X nuclear receptor activity' 
- [ ] ([lithocholic acid receptor activity](http://purl.obolibrary.org/obo/GO_0038186)): add parent: [nuclear receptor activity](http://purl.obolibrary.org/obo/GO_0004879) (please also keep the signaling receptor activity parent)  (see for example https://pubmed.ncbi.nlm.nih.gov/12016314  and https://pubmed.ncbi.nlm.nih.gov/11248085/ )
- [ ] check references for both terms
- [ ] add the new PubMed you cite (PMID:36409000) as a reference to both terms

**BP**

- [ ] obsolete grouping class [bile acid signaling pathway](http://purl.obolibrary.org/obo/GO_0038183) (used inconsistently) with CONSIDER [cell surface bile acid receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038184) and [intracellular bile acid receptor signaling pathway](http://purl.obolibrary.org/obo/GO_0038185)
- [ ] [cell surface bile acid receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038184) >> change label to 'adenylate cyclase-activating G protein-coupled bile acid receptor signaling pathway'
- [ ] >> move as a child of [adenylate cyclase-activating G protein-coupled receptor signaling pathway](http://purl.obolibrary.org/obo/GO_0007189)
- [ ] >> fix definition like that of siblings
- [ ] >> add reference PMID:36409000


I hope this is clearer! 

Thanks, Pascale

**Comment by raymond91125** [comment]
lithocholic acid receptor activity, by its definition and references cited, actually is a grouping term of vitamin D receptor (PMID:12016314) and G-protein coupled receptor (PMID:12419312).
PMID:35415765 and PMID:32480317 are more recent review articles.

**Comment by raymond91125** [comment]
Note that PMID:32480317 Table 1 shows that there are cell surface bile acid receptors other than TGR5 (adenylate cyclase pathway), e.g.  S1PR2, PMID:28120434, muscarinic receptor, PMID:12924634.

**Comment by pgaudet** [comment]
I think the muscarinic receptor is [G protein-coupled acetylcholine receptor signaling pathway involved in heart process](http://purl.obolibrary.org/obo/GO_0086093)

> Do we want to retain a grouping BP term? 

For bile acid receptor? If this is what you're asking, no; this is too confusing; people should be able to pick which pathway it is.

**Comment by raymond91125** [comment]
@pgaudet I understand that your desire to reduce inconsistent annotations by obsoleting bile acid receptor activity and bile acid signaling pathway. However, once they are gone, the two general types of bile acid receptor/response will then not be linked together in the ontology. Wouldn't this be a potential issue for enrichment analyses by reducing recall? A more conservative approach would be to make these two terms do_not_annotate and request that annotations be moved to terms more specific. Would that work?

**Comment by pgaudet** [comment]
Hi, 

I made the changes suggested above, added referenced from the comment https://github.com/geneontology/go-ontology/issues/26958#issuecomment-1942924038

I kept the BP 'bile acid signaling pathway' to group both pathway (and the MFs), but made that term 'do not annotate'. 

I have added logical definitions for the signaling pathway terms: 
GO:0038184  adenylate cyclase-activating G protein-coupled bile acid receptor signaling pathway
is_a adenylate cyclase-activating G protein-coupled receptor signaling pathway
and starts_with GO:0038182 ! G protein-coupled bile acid receptor activity

GO:0038185  intracellular bile acid receptor signaling pathway
is_GO:0141193 ! nuclear receptor signaling pathway 
and starts_with GO:0038186 ! bile acid nuclear receptor activity 

Thanks, Pascale
